compose.yaml

```diff
@@ -406,6 +406,64 @@ services:
     networks:
       - flink
 
+  iceberg-rest:
+    image: tabulario/iceberg-rest
+    container_name: iceberg-rest
+    networks:
+      iceberg:
+    ports:
+      - 8181:8181
+    environment:
+      - AWS_ACCESS_KEY_ID=admin
+      - AWS_SECRET_ACCESS_KEY=password
+      - AWS_REGION=us-east-1
+      - CATALOG_WAREHOUSE=s3://warehouse/
+      - CATALOG_IO__IMPL=org.apache.iceberg.aws.s3.S3FileIO
+      - CATALOG_S3_ENDPOINT=http://iceberg-minio:9000
+    depends_on:
+      - iceberg-minio
+      - iceberg-mc
+
+  iceberg-minio:
```

Comment:
Can you try to reuse the existing `minio` image?

Reply:
Yes definitely. Did not spend time on this as this PR is exploratory. Will reuse the existing `minio` image once we have a PR that is mergeable.